templates/cluster-api-helm/calico-dual-stack.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: addons.cluster.x-k8s.io/v1alpha1
```

Comment:
thoughts on moving these yaml manifests to the existing addons/ folder under templates?

Reply:
I think we discussed this several months ago and decided to use a separate folder since the files in that folder already use the same names and also include CRD definitions.

Reply:
Thoughts on moving the cluster-api-helm folder as a sub folder of the addons/ folder?

Reply:
Sure, I think that would be fine.